react/features/deep-linking/components/DeepLinkingDesktopPage.web.tsx

```diff
@@ -134,6 +143,18 @@ const DeepLinkingDesktopPage: React.FC<WithTranslation> = ({ t }) => {
                             : t(`${_TNS}.descriptionWithoutWeb`, { app: desktop?.appName })
                     }
                 </div>
+                <div className = { styles.descriptionLabel }>
+                    {
+                        t(`${_TNS}.noMobileApp`)
```

Comment:
Let's use a different translation key, in case we ever change the wording to be more mobile specific. `.noDesktopApp` perhaps.